test/defu.test.ts

```diff
@@ -96,6 +96,32 @@ describe('defu', () => {
     })
   })
 
+  it('should merge types of more than two objects', () => {
+    interface SomeConfig { foo: string }
+    interface SomeOtherConfig { bar: string[] }
+    interface ThirdConfig { baz: number[] }
+    expectTypeOf(defu({} as SomeConfig, {} as SomeOtherConfig, {} as ThirdConfig))
+      .toEqualTypeOf<SomeConfig & SomeOtherConfig & ThirdConfig>()
```

Comment:
Can we please make tests more explicit of what (final) type we expect?

Reply:
if you think it is helpful